README.md

```diff
@@ -72,35 +102,38 @@ $ docker run -d -it -p 8000:8000 -p 8001:8001 -p 8002:8002 \
      -e MARKLOGIC_INIT=true \
      -e MARKLOGIC_ADMIN_USERNAME=<insert admin username> \
      -e MARKLOGIC_ADMIN_PASSWORD=<insert admin password> \
-     PLACEHOLDER-FOR-DOCKER-IMAGE:DOCKER-TAG
+     store/marklogicdb/marklogic-server:10.0-8.1-centos-1.0.0-ea2
 ```
+Above command will start a docker container running MarkLogic Server and dind the given docker volume to it.
 
 ## Configuration
 
-MarkLogic Docker containers are configured via a set of environment variables.
+MarkLogic Server Docker containers are configured via a set of environment variables.
 
 
 | env var                       | value                           | required                          | default   | description                                        |
 | ------------------------------- | --------------------------------- | ----------------------------------- | ----------- | ---------------------------------------------------- |
-| MARKLOGIC_INIT                | true                            | no                                | <br/>     | when set to true, will initialize server           |
+| MARKLOGIC_INIT                | true                            | no                                | <br/>     | when set to true, will initialize MarkLogic           |
 | MARKLOGIC_ADMIN_USERNAME      | jane_doe                        | required if MARKLOGIC_INIT is set | n/a       | set MarkLogic admin user                           |
```

Comment:
I think MarkLogic in description column should be MarkLogic Server. e.g MarkLogic Server admin password.

Reply:
0d4b25a Updated